Content.Server/Silicons/Laws/SiliconLawSystem.cs

```diff
@@ -212,6 +213,7 @@ private void EnsureEmaggedRole(EntityUid uid, EmagSiliconLawComponent component)
             return;
 
         _roles.MindAddRole(mindId, new SubvertedSiliconRoleComponent { PrototypeId = component.AntagonistRole });
+        _roles.MindPlaySound(mindId, component.EmaggedSound);
```

Comment:
this should probably just be in the code for getting emagged. This sound effect is like 20 seconds long and could be easily spammed by taking an MMI in and out of a chassis.

Reply:
Good observation, despite coding for it I haven't 
played around with borgs yet.
I'll look into it later and make sure it's unspammable.
THOUGH I _would_ like to get https://github.com/space-wizards/space-station-14/pull/21484 merged
and do it that way mayb.